### PR TITLE
Add backdrop

### PR DESCRIPTION
**Pull request by sanidhya-saraswat**
Closes https://github.com/tom-james-watson/breaktimer-app/issues/129

Adding a optional backdrop feature to the popup notification.

Quick Note:

- When the backdrop is **disabled**, the app works just as before and you **will** be able to touch/interact with the apps behind breaktimer-app
- When the backdrop is **enabled**, you **won't** be able to touch/interact with the apps behind breaktimer-app.

### CODE REVIEW

**Comment by tom-james-watson** [comment]
Hello! Thanks for your great contribution. I have some changes I've made on top of your changes that clean up a few edge cases and add some deeper integration. See https://github.com/sanidhya-saraswat/breaktimer-app/pull/1. Check that out and merge it and then this should be ready to merge.

Great work!

**Comment by sanidhya-saraswat** [comment]
Wait!  Don't merge yet. Seems like a **bug** has generated in the new code. When there is no backdrop, the **alwaysOnTop** seems to not work

File: app/main/lib/windows.ts [hunk 1]
```diff
@@ -107,7 +109,10 @@ export function createBreakWindows(): void {
       },
     });
 
-    breakWindow.setVisibleOnAllWorkspaces(true);
+    breakWindow.setVisibleOnAllWorkspaces(true, { visibleOnFullScreen: true });
+    breakWindow.setAlwaysOnTop(true, "normal");
```
Comment (sanidhya-saraswat) [inline app/main/lib/windows.ts:113]: @tom-james-watson , this is creating a bug. Removing the "normal" option fixes it. (Tested on Mac)

Comment (sanidhya-saraswat) [reply app/main/lib/windows.ts]: fixed.

**Comment by sanidhya-saraswat** [comment]
Fixed. Please check once on local. Ready to merge from my end.

**Comment by tom-james-watson** [comment]
Good catch! Thanks for the awesome new feature. This is actually working so well now that what I'm left wondering is whether I should just turn this on by default. I guess it's safer to just leave it as optional for now in case there are any bugs with it. I'll make a new release soon that includes this.

**Pull request merged**